playbooks/cnf/deploy-run-cnf-tests-script.yaml

```diff
@@ -177,30 +177,14 @@
       args:
         chdir: "{{ cnf_test_dir }}/{{ cnftests_git_dest }}"
 
-    - name: List cnf test image versions
-      changed_when: false
-      ansible.builtin.shell:
-        cmd: |
-          set -o pipefail
-          brew list-builds --package=cnf-tests-container -r --state=COMPLETE | grep "v{{ version }}"
-      register: cnf_tests_output
-
-    - name: List dpdk-base image versions
-      changed_when: false
-      ansible.builtin.shell:
-        cmd: |
-          set -o pipefail
-          brew list-builds --package=dpdk-base-container -r --state=COMPLETE --owner=exd-cpaas-bot-prod | grep "v{{ version }}"
-      register: dpdk_base_output
-
     - name: Set cnf_tests_version and dpdk_base_version facts
       ansible.builtin.set_fact:
-        cnf_tests_version: "{{ cnf_tests_output.stdout_lines[0].split(' ')[0].split('-v')[-1] }}"
-        dpdk_base_version: "{{ dpdk_base_output.stdout_lines[0].split(' ')[0].split('-v')[-1] }}"
+        cnf_tests_version: "{{ version }}"
+        dpdk_base_version: "{{ version }}"
```

Comment:
OPTIONAL: if they are both the same version I don't recommend using different vars